### PR TITLE
Add Jaeger Critical Path analysis - LFX 2023-02

### PR DESCRIPTION
**Pull request by yurishkuro**

### CODE REVIEW

File: programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md
Comment (aliok) [inline programs/lfx-mentorship/2023/02-Jun-Aug/project_ideas.md:27]: While I would appreciate the addition of toc, last GSoC term showed us that necessity of maintaining the toc causes a confusion for people when they are adding their ideas here.

Given that it is the last day of idea proposals, we can still keep it though. I would leave it to @nate-double-u

**Review by aliok — COMMENTED** [review]
LGTM

Thanks @yurishkuro 

I will wait until @nate-double-u 's review to merge

**Review by nate-double-u — APPROVED** [review]
Thanks for this @yurishkuro!

**Pull request merged**
